src/transport.ts

```diff
@@ -19,53 +19,93 @@ import * as sdp from './sdp.js'
 import { WebRTCStream } from './stream.js'
 
 const log = logger('libp2p:webrtc:transport')
+
+/**
+ * The time to wait, in milliseconds, for the data channel handshake to complete
+ */
 const HANDSHAKE_TIMEOUT_MS = 10000
-const WEBRTC_CODE: number = 280
-const CERTHASH_CODE: number = 466
 
+/**
+ * Created by converting the hexadecimal protocol code to an integer.
+ *
+ * {@link https://github.com/multiformats/multiaddr/blob/master/protocols.csv}
+ */
+export const WEBRTC_CODE: number = 280
+
+/**
+ * Created by converting the hexadecimal protocol code to an integer.
+ *
+ * {@link https://github.com/multiformats/multiaddr/blob/master/protocols.csv}
+ */
+export const CERTHASH_CODE: number = 466
+
+/**
+ * The peer for this transport
+ */
+// @TODO(ddimaria): seems like an unnessary abstraction, consider removing
 export interface WebRTCTransportComponents {
   peerId: PeerId
 }
 
 export class WebRTCTransport implements Transport {
+  /**
+   * The peer for this transport
+   */
   private readonly components: WebRTCTransportComponents
 
   constructor (components: WebRTCTransportComponents) {
     this.components = components
   }
 
+  /**
+   * Dial a given multiaddr
+   */
   async dial (ma: Multiaddr, options: WebRTCDialOptions): Promise<Connection> {
     const rawConn = await this._connect(ma, options)
     log(`dialing address - ${ma.toString()}`)
     return rawConn
   }
 
+  /**
+   * Create transport listeners
+   */
   createListener (options: CreateListenerOptions): Listener {
     throw unimplemented('WebRTCTransport.createListener')
```

Comment:
@John-LittleBearLabs is there a plan to implement this?

Reply:
No, this is not going to be supported for browsers any time soon until the browser-to-browser spec is  ready.

Reply:
Thanks @ckousik, I made some comments in the README about this when discussing the the implementation of the `Transport` interface:

> Browsers can only `dial`, so `listen` is not supported.